src/pages/case-studies.js

```diff
@@ -1,47 +1,73 @@
 import React from 'react';
 import { graphql } from 'gatsby';
-import { GatsbyImage } from 'gatsby-plugin-image';
 import get from 'lodash/get';
 
 import { SEO, Page, TextLink as Link } from 'components';
 import { Attribution, TitleAndDescription, ListHeader } from 'components/article';
 
+const LogoImage = ({ caseStudy, backgroundColor }) => {
+  const images = get(caseStudy, 'logo.gatsbyImageData.images');
+  const className = 'h-48 w-full object-contain';
+  if (!images) return <div className={className} />
+
+  return (
+    <img
+      className={className}
+      style={{ backgroundColor }}
+      srcSet={images.sources[0].srcSet}
+      sizes={images.sources[0].sizes}
+      src={images.fallback.src}
+      alt={caseStudy.logo.title}
+    />
+  );
+};
+
 const CaseStudySummary = ({ study }) => {
-  const logoBackgroundColor = get(study, 'frontmatter.logo.backgroundColor', null);
+  const logoBackgroundColor = get(study, 'logoBackgroundColorHex', null);
+
+  // This will be cleaned up once the remaining content pieces (excluding docs) are migrated
+  // to Contentful.
+  const shimmedCaseStudy = {
+    frontmatter: {
+      title: study.title,
+      description: study.description.childMarkdownRemark.rawMarkdownBody,
+      author: study.author,
+      date: study.date,
+    },
+
+    fields: {
+      slug: `/case-studies/${study.slug}/`,
+    },
+
+    timeToRead: study.body.childMarkdownRemark.timeToRead,
+  };
```

Comment:
This shimming will hopefully be cleaned up when I get a chance to move /backstage/plugins into Contentful.